.changeset/wicked-ties-walk.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react': patch
```

Comment:
```suggestion
'@primer/react': minor
```

Sorry if I missed this on the other ones! Generally when we're adding in behavior we'll use `minor` here.

Reply:
Omg you're right, I had `minor` on the other ones but goofed on this one.